src/app/containers/footer/footer.component.spec.ts

```diff
@@ -15,6 +15,8 @@ describe('FooterComponent', () => {
 
     let testHostComponent: TestDummyHostComponent;
     let testHostFixture: ComponentFixture<TestDummyHostComponent>;
+    let el: DebugElement;
```

Comment:
these variables don't appear to be used

Reply:
They are used. testHostComponent in line 36, testHostFixture - line 35 and 36, variable el is now removed